rq/queue.py

```diff
@@ -387,6 +387,7 @@ def enqueue_at(self, datetime, func, *args, **kwargs):
         job = self.create_job(func, *args, **kwargs)
         registry = ScheduledJobRegistry(queue=self)
         with self.connection.pipeline() as pipeline:
+            job.set_status(JobStatus.SCHEDULED, pipeline=pipeline)
```

Comment:
Instead of changing job status to scheduled here, we should create the job with `scheduled` as status in line 387 so we don't incur an extra Redis hit.

Reply:
create_job func has no status param, and inside create_job job creating via job.create 'queued' status was hardcoded. So i will add new arg named 'status' into create_job which use 'queued' status as default. Thats will be ok?